exasol_sphinx_github_pages_generator/cli.py

```diff
@@ -92,6 +91,7 @@ def _main():
     except Exception as ex:
         if DEBUG:
             Console.stderr(traceback.format_exc())
+            print(sys.exc_info()[2])
```

Comment:
is this a debug output? if yes, please remove, otherwise make more clear what you want to output